app.py

```diff
@@ -14,16 +13,15 @@
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
 
-# Load environment variables from .env file
-load_dotenv()
```

Comment:
load_dotenv() need to be executed once. Otherwise, the .env file isn't loaded

Reply:
mybad yeah